src/main/java/dropwizard/springboot/dropwizard/to/springboot/controller/RestController.java

```diff
@@ -15,4 +27,50 @@ public class RestController {
     public String displayName(@RequestParam(name="name", required = false, defaultValue = "Stranger") String name){
         return "Hello! "+name;
     }
+
+    /**
+     * Endpoint to create/save a user
+     * @param userDto, takes userDto fields required to create and save user
+     * @return returns the entity of saved user
+     */
+    @PostMapping("/saveUser")
+    public UserEntity saveUser(@RequestBody UserDto userDto){
+        return userCrudService.saveUser(userDto);
+    }
+
+    /**
+     * Endpoint to update an existing user from its userId field
+     * @param userId, required field to identify the user which needs to be updated
+     * @param userDto, the fields which need to be updated for an id
+     * @return returns the updated userEntity with updated fields
+     */
+    @PutMapping("/updateUser")
+    public UserEntity updateUser(@RequestParam(name="userId", required = true)Integer userId,
+                                 @RequestBody UserDto userDto){
+        return userCrudService.updateUser(userId, userDto);
+    }
+
+    /**
+     * Endpoint to get a particular user from userId
+     * @param userId, required field to identify the user which needs to be fetched
+     * @return returns the fetched userEntity
+     */
+    @GetMapping("/getUser")
+    public UserEntity getUser(@RequestParam(name="userId", required = true)Integer userId){
+        return userCrudService.getUser(userId);
+    }
+
+    /**
+     * Endpoint to get all existing users
+     * @return list of userEntity
+     */
+    @GetMapping("/getAllUsers")
+    public List<UserEntity> getAllUsers(){
+        return userCrudService.getAllUsers();
+    }
+
+    @DeleteMapping("/deleteUser")

```

Comment:
If your using `@Delete` then delete in endpoint is not required, as Delete means deleting a user